CMakeLists.txt

```diff
@@ -61,16 +64,27 @@ set(CMAKE_THREAD_PREFER_PTHREAD YES)
 # Required libs.
 set(LIBS)
 
-find_package(Bfd REQUIRED)
 find_package(Threads REQUIRED)
-
-list(APPEND LIBS Bfd::Bfd Threads::Threads m)
+list(APPEND LIBS Threads::Threads m)
 
 # Optional libs.
-find_package(Valgrind)
-if (Valgrind_FOUND)
-  set(HAVE_VALGRIND 1)
-  list(APPEND LIBS Valgrind::Valgrind)
+set(FIND_MODULES)
+
+if (ENABLE_STACKTRACE_BFD)
+  find_package(Bfd REQUIRED)
+  if (Bfd_FOUND)
+    list(APPEND LIBS Bfd::Bfd)
+    list(APPEND FIND_MODULES cmake/FindBfd.cmake)
+  endif ()
+endif ()
+
+if (ENABLE_VALGRIND)
+  find_package(Valgrind)
+  if (Valgrind_FOUND)
+    set(HAVE_VALGRIND 1)
```

Comment:
Is it necessary? Since you have `Valgrind_FOUND` can't you use it instead?

Reply:
I must declare a new variable for a C file:
https://github.com/xcp-ng/xcp-ng-generic-lib/blob/master/src/coroutine.c#L34